frontend/packages/erd-core/src/components/ERDRenderer/ERDRenderer.tsx

```diff
@@ -11,10 +11,7 @@ import { toggleLogEvent } from '@/features/gtm/utils'
 import { useVersion } from '@/providers'
 import { useDBStructureStore, useUserEditingStore } from '@/stores'
 import { CardinalityMarkers } from './CardinalityMarkers'
-// biome-ignore lint/nursery/useImportRestrictions: Fixed in the next PR.
-import { MobileToolbar } from './ERDContent/Toolbar'
-// biome-ignore lint/nursery/useImportRestrictions: Fixed in the next PR.
-import { DesktopToolbar } from './ERDContent/Toolbar'
+import { Toolbar } from './ERDContent'
```

Comment:
It feels a bit off. Maybe elevating the `Toolbar` directory to the same layer as `ERDContent` (moving it one level up) would be more natural. It's a simple change, but it would result in a large diff, so I'd prefer to handle it in a separate PR. 🙏

Reply:
> I'd prefer to handle it in a separate PR. 🙏

https://github.com/liam-hq/liam/pull/621